src/OrbitGl/MockTextRenderer.cpp

```diff
@@ -32,7 +32,13 @@ void MockTextRenderer::AddText(const char* text, float x, float y, float z,
   }
   float text_height = GetStringHeight(text, formatting.font_size);
 
-  float real_start_x = formatting.halign == HAlign::Left ? x : x - text_width;
+  float real_start_x = x;
```

Comment:
I would prefer either using a switch here or modify the switch below to be consistent, we are basically doing the same in 2 different ways.

Reply:
Ah, true. I was deciding against a switch because we're not covering all cases, but it makes sense to keep this consistent.